dev/TeachingTip/TeachingTip.idl

```diff
@@ -128,6 +136,7 @@ unsealed runtimeclass TeachingTip : Windows.UI.Xaml.Controls.ContentControl
     event Windows.Foundation.TypedEventHandler<TeachingTip, Object> CloseButtonClick;
     event Windows.Foundation.TypedEventHandler<TeachingTip, TeachingTipClosingEventArgs> Closing;
     event Windows.Foundation.TypedEventHandler<TeachingTip, TeachingTipClosedEventArgs> Closed;
+    event Windows.Foundation.TypedEventHandler<TeachingTip, TeachingTipOpenedEventArgs> Opened;
```

Comment:
Same here, you can find an example to mark a single property as preview [here](https://github.com/microsoft/microsoft-ui-xaml/blob/9052972906c8a0a1b6cb5d5c61b27d6d27cd7f11/dev/NumberBox/NumberBox.idl#L61-L64).